CHANGELOG.rst

```diff
@@ -5,13 +5,27 @@
 
 Renku ``0.21.0`` brings the ability to handle expiring Gitlab tokens in user sessions and various bug fixes. 
 
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+* 🎨 **UI**: Improve the layout of the project creation, session start, and file browser pages.
+
+**🐞 Bug Fixes**
+
+* 📃 **UI**: Update broken links to local projects and documentation, and add more links to useful resources.
```

Comment:
Where were these broken links, and where are the new links to useful resources?

Reply:
Scattered in many places -- workflows, notebooks, other project pages... Here is the full list https://github.com/SwissDataScienceCenter/renku-ui/pull/2199/files

I tried to condense a few minor fixes here. Since they are minor, I didn't want to write 3 separate sentences. But here are the actual changes:
1. Update broken links (result of documentation changes) https://github.com/SwissDataScienceCenter/renku-ui/pull/2199/files
2. Add link to what is an S3 storage object https://github.com/SwissDataScienceCenter/renku-ui/pull/2207/files
3. Fix link to local projects https://github.com/SwissDataScienceCenter/renku-ui/pull/2209/files